eng/Versions.props

```diff
@@ -11,8 +11,8 @@
   </PropertyGroup>
   <!-- Repo Version Information -->
   <PropertyGroup>
-    <VersionPrefix>8.0.300</VersionPrefix>
-    <WorkloadsFeatureBand>8.0.300</WorkloadsFeatureBand>
+    <VersionPrefix>8.0.400</VersionPrefix>
+    <WorkloadsFeatureBand>8.0.400</WorkloadsFeatureBand>
```

Comment:
I think all of the 8.0 workloads are 8.0.100 band. Did this get incorrectly updated in 2xx and 3xx? Is it even used?

Reply:
Well, I tried searching for all references to WorkloadsFeatureBand in SDK and Installer, and this line was the only one that came up, so I think the answer is no? I can move it back to 8.0.100 or remove it if you'd like.